gensim/models/word2vec.py

```diff
@@ -26,39 +28,64 @@
+The word vectors are stored in a KeyedVectors instance in model.wv. This separates the read-only word vector lookup operations in KeyedVectors from the training code in Word2Vec.
+
+  >>> model.wv['computer']  # numpy vector of a word
+  array([-0.00449447, -0.00310097,  0.02421786, ...], dtype=float32)
+
+The word vectors can also be instantiated from an existing file on disk in the word2vec C format as a KeyedVectors instance::
+
+NOTE: It is impossible to continue training the vectors loaded from the C format because the binary tree is missing.
 
-  >>> model = Word2Vec.load_word2vec_format('/tmp/vectors.txt', binary=False)  # C text format
-  >>> model = Word2Vec.load_word2vec_format('/tmp/vectors.bin', binary=True)  # C binary format
+  >>> from gensim.models.keyedvectors import KeyedVectors
+  >>> word_vectors = KeyedVectors.load_word2vec_format('/tmp/vectors.txt', binary=False)  # C text format
+  >>> word_vectors = KeyedVectors.load_word2vec_format('/tmp/vectors.bin', binary=True)  # C binary format
 
-You can perform various syntactic/semantic NLP word tasks with the model. Some of them
+
+You can perform various NLP word tasks with the model. Some of them
 are already built-in::
 
-  >>> model.most_similar(positive=['woman', 'king'], negative=['man'])
+  >>> model.wv.most_similar(positive=['woman', 'king'], negative=['man'])
   [('queen', 0.50882536), ...]
 
-  >>> model.doesnt_match("breakfast cereal dinner lunch".split())
+  >>> model.wv.most_similar_cosmul(positive=['woman', 'king'], negative=['man'])
+  [('queen', 0.71382287), ...]
+
+
+  >>> model.wv.doesnt_match("breakfast cereal dinner lunch".split())
   'cereal'
 
-  >>> model.similarity('woman', 'man')
+  >>> model.wv.similarity('woman', 'man')
   0.73723527
 
-  >>> model['computer']  # raw numpy vector of a word
-  array([-0.00449447, -0.00310097,  0.02421786, ...], dtype=float32)
+Probability of a text under the model::
+
+  >>> model.score(["The fox jumped over a lazy dog".split()])
+  0.2158356
+
+Correlation with human opinion on word similarity::
+
+  >>> model.wv.evaluate_word_pairs(os.path.join(module_path, 'test_data','wordsim353.tsv'))
+  0.51, 0.62, 0.13
+
+And on analogies::
+
+  >>> model.wv.accuracy(os.path.join(module_path, 'test_data', 'questions-words.txt'))
 
 and so on.
 
+
 If you're finished training a model (=no more updates, only querying), you can do
 
-  >>> model.init_sims(replace=True)
+  >>> model.delete_temporary_training_data(replace_word_vectors_with_normalized=True)
 
 to trim unneeded model memory = use (much) less RAM.
 
 Note that there is a :mod:`gensim.models.phrases` module which lets you automatically
 detect phrases longer than one word. Using phrases, you can learn a word2vec model
 where "words" are actually multiword expressions, such as `new_york_times` or `financial_crisis`:
 
->>> bigram_transformer = gensim.models.Phrases(sentences)
+>>> bigram_transformer = gensim.models.Phraser(gensim.models.Phrases(sentences))
```

Comment:
Personally I might not recommend all users prefer `Phraser` without understanding the extra steps it requires, because of the extra time of the reduction-pass, and the fact it throws out some info (in `Phrases`) that was expensive to collect and allow experimentation with different count/threshold values.